### DISCUSSION TITLE
Why it doesn't use API of some blockchain explorers out there?

### DISCUSSION THREAD

**Discussion by ilearnio** [btcpayserver/btcpayserver#4336] [Ideas / Feature Requests]
Instead of watching for changes on the blockchain with NBXplorer and Bitcoin Core it could have utilize some APIs, like https://www.blockchain.com/explorer/api (I'm pretty sure there's a lot of other alternatives). This way it will not be needed to sync the blockchain to the hard drive and thus it would help to reduce server costs, reduce server setup time (no need to wait for ~2 days while blockchain syncing), reduce overall setup complexity. WDYT?

**Comment by Kukks** [comment]
Btcpayserver is a project that focuses on being self reliant with
minimizing dependencies on third parties.

What happens when that party becomes malicious and starts lying to you
about payments and transactions, even collecting data on your wallet and
business activity and sells it off to the highest bidder?

Trusted third parties are security holes:
https://nakamotoinstitute.org/trusted-third-parties

Running a bitcoin full node helps the network stay censorship resistant and
helps you remain in control of your money.

Btcpayserver was born in an adverserial environment where malicious
entities attempted to hijack the bitcoin network. Running a full node was
key to not allowing these entities to be successful.

So yes, we might make you spend a few extra bucks on server costs through
the compulsory bitcoin node integration, but it's worth it beyond any doubt.
